types/chrome/index.d.ts

```diff
@@ -12586,6 +12591,8 @@ declare namespace chrome.sidePanel {
 
     export function getPanelBehavior(callback?: (behavior: PanelBehavior) => void): Promise<PanelBehavior>;
 
+    export function open(options: OpenOptions, callback?: () => void): Promise<void>;
```

Comment:
Hey @matcarlson, I'm more than happy to include this change, but I do think it's worth mentioning this is the case for all methods, not just `open`. Should I
a) Be consistent with the existing definitions
b) Update the other method definitions
c) Adopt only the change as suggested above

Reply:
Thanks @TuckerWhitehouse -- Ideally I'd prefer to update the other methods, within reason - this would be a lot of work to do across all of the `chrome.*` namespace. 

Note: You may want to hold off for a bit -- Slightly after I sent this review I also started to review #67146, which basically does this for all the methods in `chrome.sidePanel`. Feel free to update it if you'd like, though. #67146 has also been pending for a few weeks so the author may not update their PR right away. 

More ideally we'd refactor the whole namespace to make it consistent, but I'm not able to dedicate the time to either do it or review a 12k-line change in a single go. 

I had looked into translating the types from Chrome sources a while back, so they'd be easier to keep up-to-date, but I didn't find a way to do it that seemed sustainable and repeatable. I'm open to any ideas you may have  :1st_place_medal: